webpack.config.client.js

```diff
@@ -89,7 +97,7 @@ module.exports = ({
         maxSize: 245760, // 240kb
         cacheGroups: {
           default: false,
-          vendors: false,
+          defaultVendors: false,
```

Comment:
From [the update-outdated-options section](https://webpack.js.org/migrate/5/#update-outdated-options) of the webpack upgrade docs 

> optimization.splitChunks.cacheGroups.vendors → optimization.splitChunks.cacheGroups.defaultVendors